README.md

```diff
@@ -40,9 +40,10 @@ Datadog APM for Go is built upon dependencies defined in specific versions of th
 <!-- NOTE: When updating the below section ensure you update the minimum supported version listed in the public docs here: https://docs.datadoghq.com/tracing/setup_overview/setup/go/?tab=containers#compatibility-requirements -->
```

Comment:
Is this in a separate PR?

Reply:
Ah it will / should be, thanks for the reminder since I totally forgot

Reply:
https://github.com/DataDog/documentation/pull/14886